server_mode/README.rst

```diff
@@ -1,74 +1,82 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+.. |company| replace:: ADHOC SA
+
+.. |company_logo| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-logo.png
+   :alt: ADHOC SA
+   :target: https://www.adhoc.com.ar
+
+.. |icon| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-icon.png
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
    :alt: License: AGPL-3
 
 ===========
 Server Mode
 ===========
 
 This modules disable some functions when running databases on odoo servers with
-parameter server_mode = "some value"
+parameter server_mode = "some value". It will show a ribbon in the instance
+which text will be the value saved in server_mode paramater: if this one is
+not set then will not show any ribbon.
+
+It also disable crons for instances with server_mode value.
+
+   **NOTE** Maybe we see that is better to use activated ir.cron in a not production environment, in this case we can disable them in the odoo.conf using entrypoint.sh taking into account the environment type or any other parameter that could disable them.
 
-It also disable crons for instances with server_mode value (NOTA: tal vez veamos que nos conviene usar cron activados en no prod, en este caso podemos desactivarlos en el odoo conf usando el entrypoint para que segun el tipo o algun parametro los desactive)
 
 This module is also inherited by other modules so that you can disable
 functionalities depending on server mode. To use it:
 
 * import with: from openerp.addons.server_mode.mode import get_mode
-* use it like following:
-    * if mode() == 'test':
-    * if mode() == 'develop'
-    * if mode():
-    * ... etc
+* use it like any of the following::
+
+   if mode() == 'test':
```

Comment:
My mistake here, it would be "if get_mode()..." (not "if mode()")

Reply:
Fixed